### DISCUSSION TITLE
Why POST binding is unsupported for logout response?

### DISCUSSION THREAD

**Discussion by vladsf** [latchset/mod_auth_mellon#64] [Q&A] [answered]
Hi,
Why POST binding is unsupported for logout response?

**Comment by thijskh** [comment]
The short answer is that this is just not implemented. Using POST for logout is uncommon and also not allowed by the saml2int profile. Probably a PR would be accepted to implement it, if not too intrustive, however, given international standards its probably better in the long run if the other party gains support for the more standard HTTP-Redirect binding instead.

**Comment by thijskh** [comment, marked as answer]
A PR has been merged that implements this: https://github.com/latchset/mod_auth_mellon/pull/84. Will be part of Mellon 1.19.
